src/main/java/ch/njol/skript/Skript.java

```diff
@@ -565,45 +566,51 @@ public void run() {
 				
 				Documentation.generate(); // TODO move to test classes?
 				
-				if (logNormal())
-					info("Loading variables...");
-				final long vls = System.currentTimeMillis();
-				
-				LogHandler h = SkriptLogger.startLogHandler(new ErrorDescLogHandler() {
-					@Override
-					public LogResult log(final LogEntry entry) {
-						super.log(entry);
-						if (entry.level.intValue() >= Level.SEVERE.intValue()) {
-							logEx(entry.message); // no [Skript] prefix
-							return LogResult.DO_NOT_LOG;
-						} else {
-							return LogResult.LOG;
-						}
-					}
+				// Variables
+				Bukkit.getScheduler().scheduleSyncDelayedTask(Skript.this, () -> {
```

Comment:
Won't this allow code to be run before variables are loaded?

Reply:
No, Skript hooks works the same way.